BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -428,12 +428,13 @@ extension AppProcessor {
     private func getOtpAuthUrlRoute(url: URL) async -> AppRoute? {
         guard let scheme = url.scheme, scheme.isOtpAuthScheme else { return nil }
 
-        guard let otpAuthModel = OTPAuthModel(otpAuthKey: url.absoluteString) else {
+        let totpKeyModel = TOTPKeyModel(authenticatorKey: url.absoluteString)
+        guard case .otpAuthUri = totpKeyModel.totpKey else {
```

Comment:
@matt-livefront @fedemkr @KatherineInCode I'd love to hear your thoughts on this. I left it in to require it to be an OTP URL to keep the handling consistent. We have a few tests that expect a failure here when we send various incomplete `otpauth://` URLs.

However the TOTPKeyModel is much more forgiving and will simply treat these as other code types. So we could theoretically just remove those tests and we don't need to guard for the scheme or check to be sure it parsed to an `.otpAuthUri` type. Like I said, I left this in to keep it consistent, but I'm open to re-considering that.

Reply:
I think I'm in favor of keeping the `otpauth://` scheme check, but I like the idea of removing the `.otpAuthUri` case check and associated error. It seems better to let the user proceed rather than throwing an error if we can't parse to a `.otpAuthUri` type.

Reply:
🤔 I agree on keeping the scheme check. Now given the method `getOtpAuthUrlRoute` is explicitly for OTP auth URIs then IMO it makes sense to have the `.otpAuthUri` case check and associated error. Is there a scenario you can think of or @KatherineInCode (I know you've seen several special cases) where a `otpauth://` is passed and we cannot parse it but it's fine to let the user continue?

Reply:
I would definitely keep the check, we cannot allow TOTP creation without a key.